client/src/templates/Challenges/video.css

```diff
@@ -91,6 +86,11 @@
   border: 2px solid var(--primary-color);
 }
 
+.video-quiz-input-hidden {
+  position: absolute;
+  left: -9999px;
+}
+
```

Comment:
this can be undone, to clean the change log.

Reply:
It is present in my latest commit.

Reply:
I am not sure how this could be undone. I mistakenly removed it, and then added it again as it was necessary for the implementation.

Reply:
no worries, when you finish the PR, I will add a quick commit if that is okay with you.

Reply:
![image](https://user-images.githubusercontent.com/103327712/233772390-0830130d-4719-4cf7-93f2-2f596f05d651.png)
This looks like the expected change? Matched the line number of it from where was removed. Correct me if its wrong.
![image](https://user-images.githubusercontent.com/103327712/233772424-373bcf99-2aeb-448a-b0ba-bb201d4c2176.png)

Reply:
Oh sure, you could add it. What I was planning was to go on previous commits using checkout and remove those changes. Not sure if that is the solution, but was trying. 

You could add it, sure.